ert_shared/models/base_run_model.py

```diff
@@ -138,11 +140,9 @@ def teardown_context(self) -> None:
 
     @job_queue(None)
     def killAllSimulations(self) -> None:
```

Comment:
Can this not be completely removed? I don't find any subclasses that implement this function.. And the `LegacyTracker` that uses it is removed in this PR.

Reply:
Indeed. This basically calls `self._job_queue.kill_all_jobs()`, which the legacy ensemble also calls.